Build/Projects/MonoGame.Framework.definition

```diff
@@ -136,6 +136,7 @@
       <!-- Covers compatible GL implementations, such as ANGLE and OpenGL -->
     </Service>
     <Service Name="_XNADesignProvided">
+      <AddDefine>XNADESIGNPROVIDED</AddDefine>
```

Comment:
Yes, plus in the future if we want to add more platforms to it, all we have to do is just connect the service, no need to modify any .cs files.